client/mysqlbinlog.cc

```diff
@@ -2997,6 +3031,58 @@ static Exit_status dump_local_log_entries(PRINT_EVENT_INFO *print_event_info,
   retval= ERROR_STOP;
 
 end:
+  /* write this and previous events to progress log, if they exist */
+  if (progress_file != NULL)
+  {
+    if (retval == OK_STOP)
+    {
+      if (prev_ev_time)
+      {
+        res = localtime((time_t *)&prev_ev_time);
+        fprintf(progress_file, "# FINAL applied event: logfile %s ", logname);
+        fprintf(progress_file, "Time: %02d%02d%02d %2d:%02d:%02d",
+            res->tm_year % 100,
```

Comment:
I think we should stick with 4 digit years to not repeat the faults of the past.

Reply:
That's a good suggestion, updated in latest commit.